vignettes/contribution_model.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Contribution to admiral"
+output: 
+  rmarkdown::html_vignette
+    
+vignette: >
+  %\VignetteIndexEntry{contribution_model}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Goal: Enable users of admiral to contribute and test a contribution model for admiral.
+ 
+Our contribution model is still in a testing phase so we advise you to contact us (slack link) when an issue is created to discuss details or align expectations if you are not familiar with the admiral philosophy and programming strategy. The team will try to review the issues within the next sprint and give some initial feedback. Since we are not a 100% fully resourced software development team it might be that some issues will take longer to respond to depending on the amount of overall issues. 
+
+
+# Basics
+ * For each new contribution, the user or Data Scientist creates an issue on our project board and put it in our backlog.
```

Comment:
I would cut out the project board. I would expect issues to be directly created in the "Issues" tab on GitHub and we'll add them to the board later on.